components/Dialogs/src/MDCAlertController.m

```diff
@@ -256,47 +225,33 @@ - (void)actionButtonPressed:(id)sender {
 
 #pragma mark - UIViewController
 
+- (void)loadView {
+  self.view = [[MDCAlertControllerView alloc] initWithFrame:CGRectZero];
+  self.alertView = (MDCAlertControllerView *)self.view;
```

Comment:
everything other than instantiating `self.view` should probably be moved to `viewDidLoad`

Reply:
alertView will only ever be a casted self.view.  It is used in place of self.view throughout.
I believe it makes sense to set here so there is never a time self.view != self.alertView.

adjusting... moved to viewDidLoad.